crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/stateless/StatelessAssessmentServiceTest.java

```diff
@@ -0,0 +1,155 @@
+import uk.gov.justice.laa.crime.meansassessment.service.AssessmentCriteriaService;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CaseType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.Frequency;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.AgeRange;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.IncomeType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.OutgoingType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.StatelessOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.AssessmentCriteriaChildWeightingRepository;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.AssessmentCriteriaDetailFrequencyRepository;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.AssessmentCriteriaRepository;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.CaseTypeAssessmentCriteriaDetailValueRepository;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+@DataJpaTest
+@RunWith(SpringRunner.class)
+public class StatelessAssessmentServiceTest {
+
+    @Autowired
+    private AssessmentCriteriaRepository assessmentCriteriaRepository;
+    @Autowired
+    private AssessmentCriteriaDetailFrequencyRepository assessmentCriteriaDetailFrequencyRepository;
+    @Autowired
+    private CaseTypeAssessmentCriteriaDetailValueRepository caseTypeAssessmentCriteriaDetailValueRepository;
+    @Autowired
+    private AssessmentCriteriaChildWeightingRepository assessmentCriteriaChildWeightingRepository;
+
+    private AssessmentCriteriaService assessmentCriteriaService;
+
+    private AssessmentCriteriaChildWeightingService childWeightingService;
+
+    private StatelessAssessmentService statelessAssessmentService;
+
+    //  These age ranges give a child weighting of 1.1 - so 2.1 overall
+    private static final Map<AgeRange, Integer> childFactor11 = Map.of(
+            AgeRange.FIVE_TO_SEVEN, 1,
+            AgeRange.EIGHT_TO_TEN, 2
+    );
+
+    @Before
+    public void setUp() {
+        assessmentCriteriaService = new AssessmentCriteriaService(assessmentCriteriaRepository,
+                assessmentCriteriaDetailFrequencyRepository, caseTypeAssessmentCriteriaDetailValueRepository,
+                assessmentCriteriaChildWeightingRepository);
+
+        childWeightingService = new AssessmentCriteriaChildWeightingService();
+
+        statelessAssessmentService = new StatelessAssessmentService(assessmentCriteriaService, childWeightingService);
+    }
+
+    @Test
+    public void initialOnlyProducesEligibleWhenAtLowerThreshold() {
+        Assert.assertEquals(StatelessOutcome.ELIGIBLE, initialOutcomeForIncome(BigDecimal.valueOf(12475)));
```

Comment:
I think we should be utilising `assertJ` here to ensure consistency with the existing test classes.